core-infrastructure/src/db/Core.Database/Scripts/017-CreateBFRTable.sql

```diff
@@ -0,0 +1,13 @@
+IF NOT EXISTS(SELECT *
+              FROM INFORMATION_SCHEMA.TABLES
+              WHERE table_name = 'BFR')
+    BEGIN
+        CREATE TABLE dbo.ComparatorSet
+        (
+            RunType         nvarchar(50)  NOT NULL,
+            RunId           nvarchar(50)  NOT NULL,
+            TrustUPIN       nvarchar(6)   NOT NULL,

```

Comment:
Is there a reason why this is using TrustUPIN rather than CompanyNumber?

Reply:
We'll need to ensure that is added to the Trust table, otherwise we won't be able to link/lookup that data.

Reply:
It will need implementing, as part of the BFR work.